interconnect/core-legacy/src/it/java/de/taimos/dvalin/interconnect/core/MessageConnectorITTest.java

```diff
@@ -20,17 +20,16 @@
  * #L%
  */
 
-import java.time.Duration;
 import java.util.HashMap;
 import java.util.UUID;
 
 import javax.jms.TextMessage;
 
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
+import org.junit.AfterClass;
```

Comment:
wieso Rückbau auf Junit4?

Reply:
Merge vom Master war bei mir schiefgelaufen, da src/it nicht im Project gezogen wird. Maven konfiguriert diesen Ordner nicht als test root